react/components/ProductAssemblyOptions/ProductAssemblyOptionItemChildrenDescription.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react'
+import { useProductAssemblyItem } from '../ProductAssemblyContext'
+
+const getItemText = (item: AssemblyItem) => {
+  const deltaQuantity = item.quantity - item.initialQuantity
+  if (deltaQuantity === 0) {
+    return null
+  }
+  if (deltaQuantity < 0 && item.quantity === 0) {
+    return `No ${item.name}`
+  }
+  const sign = deltaQuantity < 0 ? '-' : '+'
+  return `${sign}${deltaQuantity}x ${item.name}`
```

Comment:
That should be handled by a Intl message

Reply:
Future